### PR TITLE
feat: added dialog component

### PR DESCRIPTION
**Pull request by GuptaEesh**
2 types of dialogs are added:

>Simple Dialog
Action Sheet

dialogue

### CODE REVIEW

File: components/dialogues.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    border-radius: 12px;
+    box-sizing: border-box;
+    box-shadow: 0 0 8px var(--primary-200);
+    display: grid;
+    grid-template-rows: repeat(3, 3rem);
+    grid-template-columns: repeat(2, 1fr);
+    height: 10.3rem;
+    padding-top: var(--size-12);
+    width: 18rem;
+
```
Comment (ZuberDunge) [inline components/dialogues.css]: you can remove extra space

**Review by ZuberDunge — APPROVED** [review]
Apart from extra spaces, everything looks fine. Good to go from my side.

File: components/dialogues.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    color: var(--primary-200);
+}
+
+.last-action-item {
+    border-radius: 10.8px;
+    height: var(--size-32);
+    margin-top: 10.8px;
+}
+@media screen and (max-width: 1240px){
+    .dialogue-container ,.action-sheet{
```
Comment (Varun-28) [inline components/dialogues.css]: you can create another media.css file and all media query of your website in that one file

File: components/dialogues.css (hunk 1 above)
Comment (GuptaEesh) [reply components/dialogues.css]: Done✅

File: components/dialogues.css (hunk 2 above)
Comment (GuptaEesh) [reply components/dialogues.css]: Your opinion is right but I added it separately just to enhance understanding and readability.
>Person reading the code that this media query is particularly for dialogues.
>Its easier to manage too as I don't have separate HTML for each component.

**Comment by GuptaEesh** [comment]
Done with 2 review comments and improved accordingly. Merging it now.

**Pull request merged**